c8run/internal/windows/windows.go

```diff
@@ -0,0 +1,67 @@
+//go:build windows
+
+package windows
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"syscall"
+)
+
+func (w *WindowsC8Run) OpenBrowser(name string) error {
+	operateUrl := "http://localhost:8080/operate/login"
+	openBrowserCmdString := "start " + operateUrl
+	openBrowserCmd := exec.Command("cmd", "/C", openBrowserCmdString)
+	openBrowserCmd.SysProcAttr = &syscall.SysProcAttr{
+		// CreationFlags: 0x08000000 | 0x00000200, // CREATE_NO_WINDOW, CREATE_NEW_PROCESS_GROUP : https://learn.microsoft.com/en-us/windows/win32/procthread/process-creation-flags
+	}
+	fmt.Println(name + " has successfully been started.")
+	openBrowserCmd.Run()
+	return nil
+}
+
+func (w *WindowsC8Run) GetProcessTree(commandPid int) []*os.Process {
+	return process_tree(int(commandPid))
+}
+
+func (w *WindowsC8Run) GetVersionCmd(javaBinaryPath string) *exec.Cmd {
```

Comment:
Jumping on the review bandwagon! 

VersionCmd would be enough from the naming perspective and more in line with Go style guides.

Prefixing methods and functions is not idiomatic in Go. 
https://go.dev/doc/effective_go#Getters

I believe you would only do that when it's actually a GET request. 
https://google.github.io/styleguide/go/decisions.html#getters

Reply:
Thanks for jumping in. More eyes is good on such a new project.
I fixed the naming of the functions in ea381f58309